servers/src/main/java/tachyon/worker/TachyonWorker.java

```diff
@@ -47,10 +56,18 @@ public static void main(String[] args) {
 
     try {
       worker.process();
+      // Start the lineage worker
+      if (LineageUtils.isLineageEnabled(WorkerContext.getConf())) {
+        lineageWorker.start();
```

Comment:
I see. That's a good call. We can refactor the web server in a future PR that separates the constructor with the configuration of `BlockDataManager`